src/rpc/mempool.cpp

```diff
@@ -683,6 +684,7 @@ static RPCHelpMan getmempoolinfo()
                 {RPCResult::Type::STR_AMOUNT, "minrelaytxfee", "Current minimum relay fee for transactions"},
                 {RPCResult::Type::NUM, "incrementalrelayfee", "minimum fee rate increment for mempool limiting or BIP 125 replacement in " + CURRENCY_UNIT + "/kvB"},
                 {RPCResult::Type::NUM, "unbroadcastcount", "Current number of transactions that haven't passed initial broadcast yet"},
+                {RPCResult::Type::BOOL, "mempoolfullrbf", "True if the mempool accepts RBF without replaceability signaling inspection"},
```

Comment:
nit: if you retouch could rename to "fullrbf", as `getmempoolinfo` should already be clear enough that the fields are about the mempool.

Reply:
modified